tests/ga/test_extension.py

```diff
@@ -2461,6 +2461,7 @@ def test_extension_error_should_be_raised_when_continue_on_update_failure_is_fal
             (first_ext, ExtensionCommandNames.DISABLE)
         )
 
+        self.assertEqual(len(first_ext.status_blobs), 1, "The first extension should not have submitted a second status.")
```

Comment:
A follow up on my previous comment, might also be worth adding if the status was actually successful rather than just checking how many times it was reported. This goes for the `second_ext.status_blobs` too.

Reply:
I think that `second_ext.status_blobs` is in fact checking for the error code returned by `disable`; a visual inspection (within the debugger) shows that this is the only status reported to the `HTTP PUT /status` endpoint.